src/drive/web/modules/filelist/File.jsx

```diff
@@ -103,6 +103,7 @@ const SelectBox = ({ withSelectionCheckbox, selected, onClick }) => (
 const FileName = ({
   attributes,
   isRenaming,
+  disabled,
```

Comment:
Hmm. A `FileName` can be `disabled` ? What does that mean? Can we find a better name?

Reply:
Yeah you're right, that's not very clear, i'll rename it.